app/pages/dashboard.vue

```diff
@@ -1,246 +1,276 @@
-                                </path>
-                            </svg>
-                        </NuxtLink>
-                        <NuxtLink :to="`/forms/${item.ulid}/edit`">
-                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor"
-                                class="w-6 h-6 hover:text-orange-500 text-gray-500 transition-colors">
-                                <path
-                                    d="M5 18.89H6.41421L15.7279 9.57627L14.3137 8.16206L5 17.4758V18.89ZM21 20.89H3V16.6473L16.435 3.21231C16.8256 2.82179 17.4587 2.82179 17.8492 3.21231L20.6777 6.04074C21.0682 6.43126 21.0682 7.06443 20.6777 7.45495L9.24264 18.89H21V20.89ZM15.7279 6.74785L17.1421 8.16206L18.5563 6.74785L17.1421 5.33363L15.7279 6.74785Z">
-                                </path>
-                            </svg>
-                        </NuxtLink>
-                        <NuxtLink :to="`/forms/${item.ulid}/submissions`">
-                            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor"
-                                class="w-6 h-6 hover:text-emerald-500 text-gray-500 transition-colors">
-                                <path
-                                    d="M13.2 12L16 16H13.6L12 13.7143L10.4 16H8L10.8 12L8 8H10.4L12 10.2857L13.6 8H15V4H5V20H19V8H16L13.2 12ZM3 2.9918C3 2.44405 3.44749 2 3.9985 2H16L20.9997 7L21 20.9925C21 21.5489 20.5551 22 20.0066 22H3.9934C3.44476 22 3 21.5447 3 21.0082V2.9918Z">
-                                </path>
-                            </svg>
-                        </NuxtLink>
-                    </div>
-                </div>
-            </li>
-        </ul>
-        <div v-else class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto mt-2">
-            <p class="text-center text-gray-500">You have not created any forms yet. <NuxtLink to="/forms/create"
-                    class="text-blue-500">Create one now</NuxtLink>
-            </p>
-        </div>
-    </div>
-    <div class="flex flex-col px-4 xl:px-6 mt-8">
-        <h1 class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto uppercase font-bold mt-1">Actions</h1>
-        <ul class="w-full xl:w-6/12 xl:px-5 lg:px-10 lg:w-full md:w-9/12 m-auto mt-2 text-center">
-            <li class="py-2 bg-emerald-700 text-white rounded hover:bg-emerald-600 transition-colors w-40 font-semibold uppercase"
-                style="font-size: smaller;">
-                <NuxtLink to="/forms/create">Create A Form</NuxtLink>
-            </li>
-        </ul>
-    </div>
-</template>
 <script setup lang="ts">
+import { DollarSignIcon, FileTextIcon, GitGraphIcon } from 'lucide-vue-next';
```

Comment:
I see the library is tree-shakeable. I assume it won't add too much weight to the bundle. That's good.

Reply:
Yeah It is